packages/mdc-notched-outline/mdc-notched-outline.scss

```diff
@@ -68,14 +69,25 @@
     }
 
     &__notch {
+      flex: 0 0 auto;
       width: auto;
+      max-width: calc(100% - #{$mdc-notched-outline-leading-width} * 2);
     }
 
     .mdc-floating-label {
       display: inline-block;
       position: relative;
       top: 17px;
       bottom: auto;
+      max-width: 100%;
+    }
+
+    //.mdc-floating-label--float-above {
```

Comment:
Is this supposed to be removed, or uncommented?

Reply:
Removed. I changed the floating label to use scrollWidth and adjusted the max-width of the notch so it should work better now (label shows ellipsis but the notch shows the rest of the label if you adjust the text area).